src/sweetalert2.js

```diff
@@ -328,15 +328,13 @@ const sweetAlert = (...args) => {
     }
 
     if (params.toast) {
-      // Closing popup by backdrop click
+      // Closing popup by internal click
       domCache.popup.onclick = (e) => {
-        if (e.target !== domCache.popup || (params.showConfirmButton || params.showCancelButton)) {
+        if ((params.showConfirmButton || params.showCancelButton || params.input)) {
```

Comment:
> Requires no buttons to be present on the toast

should we also add a check for `params.showCloseButton` here?

Reply:
@limonte ah yes we should, lemme fix that one (forgot we added the close button 👍)